src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstSide;
+    private int secondSide;
+    private int height;
+
+    public IsoscelesTrapezoid(int firstSide, int secondSide, int height, Color color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.height = height;
+        super.color = color;

```

Comment:
Direct assignment to the superclass's field is not recommended. Instead, use a constructor in the superclass to set the color.